packages/runner-ct/src/specs/spec-file.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+import { observer } from 'mobx-react'
+import State from '../lib/state'
+
+interface SpecFileProps {
+  state: State,
+  // eslint-disable-next-line
+  spec: import('./make-spec-hierarchy').SpecFile
+}
+
+export const SpecFile = observer(
+  function SpecFile ({ spec, state }: SpecFileProps) {
+    const isActive = state.runMode === 'multi'
+      ? state.multiSpecs.some((includedSpec) => includedSpec.absolute === spec.absolute)
+      : spec.name === state.spec?.name
+
+    return (
+      <li
+        key={spec.name}
+        className="spec-list-file"
+        onClick={(e) =>
+          e.shiftKey
+            ? state.addSpecToMultiMode(spec)
+            : state.setSingleSpec(spec)
+        }
+      >
+        {state.runMode === 'single'
+          ? <input className="spec-list-radio" type="radio" checked={isActive} />
```

Comment:
Should we use font-awesome to have more control over what it looks like? 
Or do we prefer the accessibility boost the radio button gives us?

Reply:
Yes, good question. But this is actually a **radio button** — why not to use a radio button to style it? :D 

I think it can be better in a few moments 

-  a11y
-  performance
- Stable Percy snapshots